dimelo/load_processed.py

```diff
@@ -81,7 +101,25 @@ def pileup_counts_from_bedmethyl(
         for row in source_tabix.fetch():
             tabix_fields = row.split("\t")
             pileup_basemod = tabix_fields[3]
-            if mod_motif in pileup_basemod and mod_coord_in_motif in pileup_basemod:
+            keep_basemod = False
```

Comment:
I'm concerned that there seems to be a bunch of duplicated code here but I'm inclined to say that this will be addressed on Thursday as part of the rest of code review.

Reply:
Yeah there totally is; wasn't immediately clear how to clean that up so I just hammered it out for now.